test/cpu_test.cpp

```diff
@@ -1893,3 +1894,59 @@ TEST(cpu_test, tensor_copy_from_different_layout)
 
     EXPECT_EQ((vector<uint8_t>{1, 4, 2, 5, 3, 6}), read_vector<uint8_t>(b));
 }
+
+// Adding this test case in cpu_test
+// because reference kernel isn't supporting intermediate
+// output types as of now
+TEST(cpu_test, quantized_conv_int32_output)
```

Comment:
If this op is being moved to core, we should put these tests in `backend_test.in.cpp` (or something like `backend_conv.in.cpp`) and add it to the manifest for backends that don't yet support it.